### PR TITLE
update multi dc persistence to akka 2.6

### PR DESCRIPTION
**Pull request by chbatey**

### CODE REVIEW

**Comment by chbatey** [comment]
We could but I thought we discussed leaving samples without any actors keeping the classic system?

File: akka-sample-persistence-dc-java/build.sbt [hunk 1]
```diff
@@ -5,11 +5,11 @@ enablePlugins(ProtobufPlugin)
 
 scalaVersion := "2.12.8"
```
Comment (patriknw) [inline akka-sample-persistence-dc-java/build.sbt]: 2.13

File: akka-sample-persistence-dc-scala/build.sbt [hunk 2]
```diff
@@ -5,11 +5,11 @@ enablePlugins(ProtobufPlugin)
 
 scalaVersion := "2.12.8"
```
Comment (patriknw) [inline akka-sample-persistence-dc-scala/build.sbt]: 2.13

File: akka-sample-persistence-dc-scala/build.sbt [hunk 3]
```diff
@@ -5,11 +5,11 @@ enablePlugins(ProtobufPlugin)
```
Comment (patriknw) [inline akka-sample-persistence-dc-scala/build.sbt]: sbt 1.3

File: akka-sample-persistence-dc-java/build.sbt [hunk 4]
```diff
@@ -33,4 +33,7 @@ libraryDependencies ++= Seq(
   "org.scalatest" %% "scalatest" % "3.0.7" % "test"
 )
 
+// transitive dependency of akka 2.5x that is brought in by addons but evicted
+dependencyOverrides += "com.typesafe.akka" %% "akka-protobuf" % AkkaVersion
+
```
Comment (patriknw) [inline akka-sample-persistence-dc-java/build.sbt]: add mvn pom while we are at it, and update run instructions in readme
template commit: https://github.com/akka/akka-samples/pull/141/commits/b54f1b608d38d4f1c49d6df2a023a8f4d256f5e5

File: akka-sample-persistence-dc-scala/build.sbt [hunk 5]
```diff
@@ -33,4 +33,7 @@ libraryDependencies ++= Seq(
   "org.scalatest" %% "scalatest" % "3.0.7" % "test"
 )
```
Comment (patriknw) [inline akka-sample-persistence-dc-scala/build.sbt]: application.conf can be simplified slightly, artery is default

Comment (patriknw) [reply akka-sample-persistence-dc-scala/build.sbt]: also update logback.xml, see example in https://github.com/akka/akka-samples/pull/135

File: akka-sample-persistence-dc-java/build.sbt (hunk 4 above)
Comment (chbatey) [reply akka-sample-persistence-dc-java/build.sbt]: :+1: added

File: akka-sample-persistence-dc-scala/build.sbt (hunk 5 above)
Comment (chbatey) [reply akka-sample-persistence-dc-scala/build.sbt]: Added in the async appender. Hadn't thought about that for STDOUT

File: akka-sample-persistence-dc-java/build.sbt [hunk 6]
```diff
@@ -33,4 +33,7 @@ libraryDependencies ++= Seq(
   "org.scalatest" %% "scalatest" % "3.0.7" % "test"
```
Comment (raboof) [inline akka-sample-persistence-dc-java/build.sbt]: 3.0.7 doesn't seem to be available for scala 2.13?
```suggestion
  "org.scalatest" %% "scalatest" % "3.0.8" % "test"
```

File: akka-sample-persistence-dc-java/build.sbt (hunk 4 above)
Comment (raboof) [inline akka-sample-persistence-dc-java/build.sbt]: sbt couldn't find the proto for me:

```suggestion
// The default is src/main/protobuf, but the maven plugin expects them here:
sourceDirectory in ProtobufConfig := (sourceDirectory in Compile).value / "proto"

```

Comment (chbatey) [reply akka-sample-persistence-dc-java/build.sbt]: Ah thanks, i changed it to do the maven plugin's default and forgot about sbt!

File: akka-sample-persistence-dc-java/README.md [hunk 7]
```diff
@@ -3,11 +3,11 @@ akka-sample-persistence-dc-java
 
 ## How to run
 
-1. In terminal 1: `sbt "runMain sample.persistence.multidc.ThumbsUpApp cassandra"`
+1. In terminal 1: `mvn exec:java -Dexec.mainClass="sample.persistence.multidc.ThumbsUpApp" -Dexec.args="cassandra"`
 
-1. In terminal 2: `sbt "runMain sample.persistence.multidc.ThumbsUpApp 2551 eu-west"`
+1. In terminal 2: `mvn exec:java -Dexec.mainClass="sample.persistence.multidc.ThumbsUpApp" -Dexec.args="22551 eu-west"`
```
Comment (raboof) [inline akka-sample-persistence-dc-java/README.md]:
```suggestion
1. In terminal 2: `mvn exec:java -Dexec.mainClass="sample.persistence.multidc.ThumbsUpApp" -Dexec.args="2551 eu-west"`
```

File: akka-sample-persistence-dc-java/README.md [hunk 8]
```diff
@@ -3,11 +3,11 @@ akka-sample-persistence-dc-java
...
 
-1. In terminal 3: `sbt "runMain sample.persistence.multidc.ThumbsUpApp 2552 eu-central"`
+1. In terminal 3: `mvn exec:java -Dexec.mainClass="sample.persistence.multidc.ThumbsUpApp" -Dexec.args="22552 eu-central"`
```
Comment (raboof) [inline akka-sample-persistence-dc-java/README.md]:
```suggestion
1. In terminal 3: `mvn exec:java -Dexec.mainClass="sample.persistence.multidc.ThumbsUpApp" -Dexec.args="2552 eu-central"`
```

**Pull request merged**